build.gradle.kts

```diff
@@ -144,9 +144,6 @@ subprojects {
     register<Test>("testFireblocksE2E") {
       useJUnitPlatform()
 
-      include("**/AnchorPlatformCustodyEnd2EndTest**")
```

Comment:
Should the `exclude(...)` be removed from the previous `test` configuration?

Reply:
I have removed the `testFireblocksE2E`